packages/hash/api/src/seed-data/seed-users.ts

```diff
@@ -38,9 +37,11 @@ const devUsers: readonly SeededUser[] = [
 export const ensureUsersAreSeeded = async ({
   graphApi,
   logger,
+  systemOrgAccountId,
 }: {
   graphApi: GraphApi;
   logger: Logger;
+  systemOrgAccountId: string;
```

Comment:
I would have liked to see `type AccountId = string` added as a drive-by in this PR considering the amount of code we're adding that's using it.

Probably not worth it now but in the future we should consider doing such things if they're backwards-compatible (which a plain string is)